src/lib/pickles_types/plonk_types.ml

```diff
@@ -414,7 +414,7 @@ module Evals = struct
     let opt_typ impl ({ lookup; runtime_tables; _ } : Opt.Flag.t Features.t)
         ~dummy:z elt =
       Opt.typ impl lookup
-        ~dummy:(dummy z ~runtime:(not (Opt.Flag.equal runtime_tables No)))
+        ~dummy:(dummy z ~runtime:(Opt.Flag.equal runtime_tables Yes))
```

Comment:
This is actually a bugfix since it changes the behavior for `Maybe`

Reply:
Interesting point